deployer/src/main/java/org/wso2/msf4j/deployer/internal/MSF4JDeployerSC.java

```diff
@@ -0,0 +1,95 @@
+/*
+* Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.wso2.msf4j.deployer.internal;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.carbon.deployment.engine.Deployer;
+import org.wso2.carbon.kernel.startupresolver.RequiredCapabilityListener;
+import org.wso2.carbon.kernel.startupresolver.StartupServiceUtils;
+import org.wso2.msf4j.MicroservicesRegistry;
+
+import java.util.Map;
+
+/**
+ * Service component for Microservices Deployer.
+ */
```

Comment:
Same as the above comment

Reply:
This is not a direct use by MSF4J end user. Therefore IMHO we don't need to add Since version

Reply:
This is mean for the developers and is a part of java doc conventions [1]

[1] http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@since